### PR TITLE
chore(SC-940): Move extensions to separate folder

### PR DESCRIPTION
**Pull request by artall64**

### CODE REVIEW

**Review by byshape — APPROVED** [review]

File: contracts/extensions/DutchAuctionCalculator.sol [hunk 1]
```diff
@@ -5,6 +5,8 @@ pragma solidity 0.8.19;
 import "@openzeppelin/contracts/utils/math/Math.sol";
 import "../interfaces/IAmountGetter.sol";
 
+/// @title A helper that implement price decay over time from max to min
```
Comment (zZoMROT) [inline contracts/extensions/DutchAuctionCalculator.sol:8]:
```suggestion
/// @title A helper that implements price decay over time from max to min
```

File: contracts/extensions/DutchAuctionCalculator.sol [hunk 2]
```diff
@@ -5,6 +5,8 @@ pragma solidity 0.8.19;
...
+/// @notice This contact is used in 1inch Fusion orders
```
Comment (zZoMROT) [inline contracts/extensions/DutchAuctionCalculator.sol:9]:
```suggestion
/// @notice This contract is used in 1inch Fusion orders
```

File: contracts/extensions/ETHOrders.sol [hunk 3]
```diff
@@ -9,7 +9,7 @@ import "@1inch/solidity-utils/contracts/OnlyWethReceiver.sol";
 import "../interfaces/IPostInteraction.sol";
 import "../OrderLib.sol";
 
-/// @title ETH limit orders contract
+/// @title Extension that will allow to create limit order that sell ETH. ETH would need to be deposited to contract
```
Comment (zZoMROT) [inline contracts/extensions/ETHOrders.sol:12]:
```suggestion
/// @title Extension that will allow to create limit order that sell ETH. ETH must be deposited into the contract
```

**Review by zZoMROT — APPROVED** [review]

**Review by galekseev — CHANGES_REQUESTED** [review]
Please, consider suggested description changes.
Perhaps, it makes sense to move OrderLib to libraries folder (discussible).

**Comment by artall64** [comment]
@zZoMROT @galekseev pls check out lates changes on comments

**Review by galekseev — APPROVED** [review]

**Pull request merged**